django/core/management/base.py

```diff
@@ -360,6 +360,9 @@ def execute(self, *args, **options):
                 translation.activate(saved_locale)
         return output
 
+    def _check_issues(self, **kwargs):
```

Comment:
I think I'd find a name like `run_checks()` more intuitive (or with a leading underscore if you like).